kubernetes_state/auto_conf.yaml

```diff
@@ -2,6 +2,10 @@ docker_images:
   - kube-state-metrics
 
 init_config:
+  # Tags are reported as set by kube-state-metrics. If you want to translate
+  # them to other tags, you can use the labels_mapper dictionary
+  #  labels_mapper:
+  #    namespace: kube_namespace
```

Comment:
i think the plan for agent 6 is to make init_config optional when it's not needed. And I think the kubernetes_state check only needs one instance, so init_config is unnecessary. If there's no particular reason to keep it here could you move this setting to the instance?